chall01/vgrankul.py

```diff
@@ -0,0 +1,50 @@
+#!/bin/python3
+import sys
+
+sos = [
+".-",
+"-...", 
+"-.-.", 
+"-..", 
+".", 
+"..-.", 
+"--.", 
+"....", 
+"..", 
+".---", 
+"-.-", 
+".-..", 
+"--", 
+"-.", 
+"---", 
+".--.", 
+"--.-", 
+".-.", 
+"...", 
+"-", 
+"..-", 
+"...-", 
+".--", 
+"-..-", 
+"-.--", 
+"--.."]
```

Comment:
I suggest declaring / assigning array like that one in a one-line like you did for letter or Bletter under.